src/stdbuf/stdbuf.rs

```diff
@@ -1,5 +1,4 @@
 #![crate_name = "stdbuf"]
-#![feature(negate_unsigned)]
```

Comment:
I haven't modified the code, just the feature gate. Or maybe I misunderstand the feature gates?

Should I undo this?